editor/register_editor_types.cpp

```diff
@@ -99,7 +99,6 @@
 #include "editor/plugins/multimesh_editor_plugin.h"
 #include "editor/plugins/navigation_link_2d_editor_plugin.h"
 #include "editor/plugins/navigation_obstacle_2d_editor_plugin.h"
```

Comment:
What about the 2D counterpart, and other navigation editor plugins (`NavigationLink2DEditorPlugin`, `NavigationPolygonEditorPlugin`).

Reply:
Doing them one after another as I find time to test them.